### ISSUE TITLE
SSL validation failed for https://sts.us-east-1.amazonaws.com/ [Errno 2] No such file or directory

### ISSUE THREAD

**Issue by jaipili1401** [aws/aws-cli#6326]
When trying to execute any aws commands I'm getting error related to SSL certification issue, so I had to add `--no-verify-ssl` to my commands every time but I'm really bothered with this issue and currently trying to get this sorted. This issue just suddenly occur to my machine without changing anything. I have reinstalled aws cli and python several times already but the issue keeps on persisting. Does anybody know how to permanently fix this?

**Comment by stobrien89** [comment]
Hi @jaipili1401 and @PulseBright, 

Thanks for the report.  Looking further into this.

**Comment by stobrien89** [comment]
Hi all,

I tested with a fresh install of `2.2.28` on Windows 10 and OS X and wasn't able to reproduce.  More information about your respective environments (i.e. OS, custom certificate bundle, etc.) would be helpful.

**Comment by PulseBright** [comment]
This is with a vanilla install of aws-cli. I did not have it previously on this computer. I've not modified the CA bundle I used in the environment variable. This is a new Win10 PC (Build 1909 - 18363.1734) and I don't have any prior environment variables or installs that could mess with it.

I've inspected the certificate on the endpoint using openssl (`v1.1.1k`):

`openssl s_client -showcerts -servername eks.ca-central-1.amazonaws.com -connect eks.ca-central-1.amazonaws.com:443`

The last certificate in the chain showed a `Verify return code: 20 (unable to get local issuer certificate)` error as well. ([pastebin](https://pastebin.com/baD8YvEj))

**Comment by jaipili1401** [comment]
> This is with a vanilla install of aws-cli installed via MSI. I did not have it previously on this computer. I've not modified the CA bundle I used in the environment variable. This is a new Win10 PC (Build 1909 - 18363.1734) and I don't have any prior environment variables or installs that could mess with it.
> 
> I've inspected the certificate on the endpoint using openssl (`v1.1.1k`):
> 
> `openssl s_client -showcerts -servername eks.ca-central-1.amazonaws.com -connect eks.ca-central-1.amazonaws.com:443`
> 
> The last certificate in the chain showed a `Verify return code: 20 (unable to get local issuer certificate)` error as well. ([pastebin](https://pastebin.com/baD8YvEj))

Thanks! this works for me, however whenever I exit my terminal I have to initialize that variable again. I'm fine doing this in the meantime but I'm looking for a solution to make my terminal work the way how it works before because this issue actually affected not only my aws cli but even git cloning or pulling packages in python repos and it really bothers me :(

**Comment by jaipili1401** [comment]
> Hi @PulseBright and @jaipili1401,
> 
> Thanks for the clarification. To work around this in the meantime, you can use the [`ca_bundle`](https://docs.aws.amazon.com/en_us/sdkref/latest/guide/setting-global-ca_bundle.html) setting in your [config file](https://docs.aws.amazon.com/sdkref/latest/guide/file-location.html) to avoid having to set `AWS_CA_BUNDLE` during each terminal session. Going to review this a bit further with the team today. @jaipili1401, are you using Windows 10 as well?

Hi @stobrien89 

Sorry for the late response, but yes I'm using Windows 10.

**Comment by stobrien89** [comment]
Thanks for the clarification, @jaipili1401.  

I reviewed with the team and we don't think anything with the CLI installation specifically is causing this, especially given that @jaipili1401 mentioned issues with git and pulling python packages as well.  A few possible culprits could be proxy configurations/ssl inspection and using Python modules like certifi (if working inside of a Python environment).  Closing, as this appears to be OS/environment specific.
